refine.bat

```diff
@@ -171,7 +171,6 @@ set REFINE_LIB_DIR=server\target\lib
 :gotLibDir
 
 if not "%REFINE_VERBOSITY%" == "" goto gotVerbosity
-set REFINE_VERBOSITY=info
```

Comment:
Removing setting info by default, otherwise the log level will be set info always.
To run default log level settings, not /v arg should be supplied